docker-compose.yml

```
@@ -50,7 +50,7 @@ services:
     volumes:
       - ./output:/home/saver/shared
   trustymail_report:
-    image: 'cisagov/trustymail_reporter:1.4.1'
+    image: 'cisagov/trustymail_reporter:1.4.2'
```

Given https://github.com/cisagov/trustymail_reporter/pull/60 should this be updated?

```suggestion
    image: 'cisagov/trustymail_reporter:1.4.3'
```

---

Yes.  I was waiting for cisagov/trustymail_reporter#60 to get approved and merged before making that changed.